Micronaut 4.3.0+ and Micronaut GCP 5.4.0 Causes Stackoverflow on Startup

### Expected Behavior

Application which starts correctly with GCP tracing enabled.

### Actual Behaviour

With an application using Micronaut GCP 5.4.0 that has Stackdriver tracing enabled will receive a stack overflow at startup.

Stacktrace:
```
Caused by: java.lang.StackOverflowError: null
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at io.micronaut.context.exceptions.BeanInstantiationException.<init>(BeanInstantiationException.java:49)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2345)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.http.client.netty.$DefaultNettyHttpClientRegistry$HttpClient0$Definition.doInstantiate(Unknown Source)
	at io.micronaut.context.AbstractInitializableBeanDefinition.instantiate(AbstractInitializableBeanDefinition.java:774)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2328)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:1089)
	at io.micronaut.context.DefaultBeanContext.createBean(DefaultBeanContext.java:1069)
	at io.micronaut.context.BeanContext.createBean(BeanContext.java:157)
	at io.micronaut.gcp.credentials.DefaultOAuth2HttpTransportFactory.<init>(DefaultOAuth2HttpTransportFactory.java:71)
	at io.micronaut.gcp.credentials.$DefaultOAuth2HttpTransportFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1813)
	at io.micronaut.context.AbstractBeanResolutionContext.findBean(AbstractBeanResolutionContext.java:112)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2163)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$DefaultGoogleCredentials0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.tracing.zipkin.$StackdriverSenderFactory$StackdriverSender1$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.$AsyncReporterConfiguration$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.tracing.zipkin.$StackdriverSenderFactory$StackdriverReporter4$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1813)
	at io.micronaut.context.AbstractBeanResolutionContext.findBean(AbstractBeanResolutionContext.java:112)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2163)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.$BraveTracerFactory$BraveTracing0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$HttpTracingFactory$HttpTracing0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$HttpTracingFactory$HttpClientHandler1$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$BraveTracingClientFilter$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2951)
	at io.micronaut.context.DefaultBeanContext.addCandidateToList(DefaultBeanContext.java:3566)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistrations(DefaultBeanContext.java:3514)
	at io.micronaut.context.DefaultBeanContext.getBeanRegistrations(DefaultBeanContext.java:3484)
	at io.micronaut.context.DefaultBeanContext.getBeansOfType(DefaultBeanContext.java:1451)
	at io.micronaut.context.AbstractBeanResolutionContext.getBeansOfType(AbstractBeanResolutionContext.java:95)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBeansOfType(AbstractInitializableBeanDefinition.java:2253)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeansOfTypeForConstructorArgument(AbstractInitializableBeanDefinition.java:1476)
	at io.micronaut.http.client.filter.$DefaultHttpClientFilterResolver$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.http.client.netty.$DefaultNettyHttpClientRegistry$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.http.client.netty.$DefaultNettyHttpClientRegistry$HttpClient0$Definition.doInstantiate(Unknown Source)
	at io.micronaut.context.AbstractInitializableBeanDefinition.instantiate(AbstractInitializableBeanDefinition.java:774)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2328)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:1089)
	at io.micronaut.context.DefaultBeanContext.createBean(DefaultBeanContext.java:1069)
	at io.micronaut.context.BeanContext.createBean(BeanContext.java:157)
	at io.micronaut.gcp.credentials.DefaultOAuth2HttpTransportFactory.<init>(DefaultOAuth2HttpTransportFactory.java:71)
	at io.micronaut.gcp.credentials.$DefaultOAuth2HttpTransportFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1813)
	at io.micronaut.context.AbstractBeanResolutionContext.findBean(AbstractBeanResolutionContext.java:112)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2163)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$DefaultGoogleCredentials0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.tracing.zipkin.$StackdriverSenderFactory$StackdriverSender1$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.$AsyncReporterConfiguration$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.tracing.zipkin.$StackdriverSenderFactory$StackdriverReporter4$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1813)
	at io.micronaut.context.AbstractBeanResolutionContext.findBean(AbstractBeanResolutionContext.java:112)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2163)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.$BraveTracerFactory$BraveTracing0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$HttpTracingFactory$HttpTracing0$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$HttpTracingFactory$HttpClientHandler1$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.tracing.brave.http.$BraveTracingClientFilter$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2951)
	at io.micronaut.context.DefaultBeanContext.addCandidateToList(DefaultBeanContext.java:3566)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistrations(DefaultBeanContext.java:3514)
	at io.micronaut.context.DefaultBeanContext.getBeanRegistrations(DefaultBeanContext.java:3484)
	at io.micronaut.context.DefaultBeanContext.getBeansOfType(DefaultBeanContext.java:1451)
	at io.micronaut.context.AbstractBeanResolutionContext.getBeansOfType(AbstractBeanResolutionContext.java:95)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBeansOfType(AbstractInitializableBeanDefinition.java:2253)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeansOfTypeForConstructorArgument(AbstractInitializableBeanDefinition.java:1476)
	at io.micronaut.http.client.filter.$DefaultHttpClientFilterResolver$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:89)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2165)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.http.client.netty.$DefaultNettyHttpClientRegistry$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.http.client.netty.$DefaultNettyHttpClientRegistry$HttpClient0$Definition.doInstantiate(Unknown Source)
	at io.micronaut.context.AbstractInitializableBeanDefinition.instantiate(AbstractInitializableBeanDefinition.java:774)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2328)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:1089)
	at io.micronaut.context.DefaultBeanContext.createBean(DefaultBeanContext.java:1069)
	at io.micronaut.context.BeanContext.createBean(BeanContext.java:157)
	at io.micronaut.gcp.credentials.DefaultOAuth2HttpTransportFactory.<init>(DefaultOAuth2HttpTransportFactory.java:71)
	at io.micronaut.gcp.credentials.$DefaultOAuth2HttpTransportFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1813)
	at io.micronaut.context.AbstractBeanResolutionContext.findBean(AbstractBeanResolutionContext.java:112)
	at io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2163)
	at io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1328)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$Definition.instantiate(Unknown Source)
	at io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2330)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2300)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2312)
	at io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3114)
	at io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)
	at io.micronaut.context.DefaultBeanContext.findOrCreateSingletonBeanRegistration(DefaultBeanContext.java:3016)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2977)
	at io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2751)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1750)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1732)
	at io.micronaut.gcp.credentials.$GoogleCredentialsFactory$DefaultGoogleCredentials0$Definition.instantiate(Unknown Source)
```

Removed a lot of the stack trace for brevity.

Is it possible it's related to: https://github.com/micronaut-projects/micronaut-gcp/pull/1014 ?

### Steps To Reproduce

Run a Micronaut 4.3.2 and Micronaut GCP 5.4.0 application with Google Stackdriver tracing enabled.

### Environment Information

GCP Cloud Run

### Example Application

_No response_

### Version

4.3.1

--- a/gcp-common/src/main/java/io/micronaut/gcp/credentials/DefaultOAuth2HttpTransportFactory.java
+++ b/gcp-common/src/main/java/io/micronaut/gcp/credentials/DefaultOAuth2HttpTransportFactory.java
@@ -26,6 +26,7 @@ import io.micronaut.context.annotation.Requires;
 import io.micronaut.core.io.buffer.ByteBuffer;
 import io.micronaut.core.type.MutableHeaders;
 import io.micronaut.core.util.StringUtils;
+import io.micronaut.core.util.SupplierUtil;
 import io.micronaut.http.*;
 import io.micronaut.http.client.BlockingHttpClient;
 import io.micronaut.http.client.HttpClient;
@@ -40,6 +41,7 @@ import java.io.IOException;
 import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.function.Supplier;
 
 /**
  * An implementation of {@link HttpTransportFactory} based upon {@link HttpClient} that can be supplied
@@ -59,7 +61,7 @@ public class DefaultOAuth2HttpTransportFactory implements HttpTransportFactory {
 
     private static final Logger LOG = LoggerFactory.getLogger(DefaultOAuth2HttpTransportFactory.class);
 
-    private final HttpClientHttpTransport transport;
+    private final Supplier<HttpClientHttpTransport> transport;
 
     /**
      * Constructor for {@code DefaultOAuth2HttpTransportFactory}.
@@ -68,13 +70,14 @@ public class DefaultOAuth2HttpTransportFactory implements HttpTransportFactory {
      * @param defaultClientConfiguration the default HTTP client configuration
      */
     public DefaultOAuth2HttpTransportFactory(BeanContext beanContext, HttpClientConfiguration defaultClientConfiguration) {
-        HttpClient httpClient = beanContext.createBean(HttpClient.class, LoadBalancer.empty(), defaultClientConfiguration);
-        this.transport = new HttpClientHttpTransport(httpClient);
+        this.transport = SupplierUtil.memoized(() -> new HttpClientHttpTransport(beanContext.createBean(HttpClient.class,
+            LoadBalancer.empty(),
+            defaultClientConfiguration)));
     }
 
     @Override
     public HttpTransport create() {
-        return this.transport;
+        return this.transport.get();
     }
 
     private static final class HttpClientHttpTransport extends HttpTransport {

--- a/gcp-tracing/build.gradle.kts
+++ b/gcp-tracing/build.gradle.kts
@@ -13,4 +13,6 @@ dependencies {
     implementation(libs.brave.opentracing)
     testAnnotationProcessor(mn.micronaut.inject.java)
     testImplementation(mn.micronaut.inject.java)
+    testImplementation(mnSerde.micronaut.serde.jackson)
+    testImplementation(testFixtures(project(":micronaut-gcp-common")))
 }

--- a/gcp-tracing/src/test/groovy/io/micronaut/gcp/tracing/zipkin/StackdriverConfigurationSpec.groovy
+++ b/gcp-tracing/src/test/groovy/io/micronaut/gcp/tracing/zipkin/StackdriverConfigurationSpec.groovy
@@ -28,5 +28,8 @@ class StackdriverConfigurationSpec extends Specification {
         then:
         noExceptionThrown()
         sender
+        
+        cleanup:
+        ctx.close()
     }
 }

--- a/gcp-tracing/src/test/groovy/io/micronaut/gcp/tracing/zipkin/StackdriverConfigurationSpec.groovy
+++ b/gcp-tracing/src/test/groovy/io/micronaut/gcp/tracing/zipkin/StackdriverConfigurationSpec.groovy
@@ -1,0 +1,32 @@
+package io.micronaut.gcp.tracing.zipkin
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.gcp.credentials.GoogleCredentialsConfiguration
+import io.micronaut.gcp.credentials.fixture.ServiceAccountCredentialsTestHelper
+import spock.lang.Issue
+import spock.lang.Specification
+import zipkin2.reporter.Sender
+
+import java.security.PrivateKey
+
+class StackdriverConfigurationSpec extends Specification {
+
+    @Issue("https://github.com/micronaut-projects/micronaut-gcp/issues/1045")
+    void "application starts successfully with stackdriver and zipkin enabled"() {
+        given:
+        PrivateKey pk = ServiceAccountCredentialsTestHelper.generatePrivateKey()
+        File serviceAccountCredentials = ServiceAccountCredentialsTestHelper.writeServiceCredentialsToTempFile(pk)
+        ApplicationContext ctx = ApplicationContext.run([
+                "gcp.project-id" : "test-project",
+                (GoogleCredentialsConfiguration.PREFIX + ".location"): serviceAccountCredentials.getPath(),
+                "tracing.zipkin.enabled": true
+        ])
+
+        when:
+        Sender sender = ctx.getBean(Sender.class)
+
+        then:
+        noExceptionThrown()
+        sender
+    }
+}

--- a/gcp-tracing/src/test/resources/logback.xml
+++ b/gcp-tracing/src/test/resources/logback.xml
@@ -1,0 +1,14 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <withJansi>false</withJansi>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+
+</configuration>

--- a/gcp-tracing/src/test/resources/logback.xml
+++ b/gcp-tracing/src/test/resources/logback.xml
@@ -1,7 +1,6 @@
 <configuration>
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <withJansi>false</withJansi>
         <encoder>
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
         </encoder>

